services/login-settings/src/types.ts

```diff
@@ -0,0 +1,28 @@
+export interface Locale {
+    locale: string
+    name: string
+    displayName?: string
+}
+
+export interface LoginSettings {
```

Comment:
Two questions:

1. Here I am just mixing everything I want to provide to the login app (e.g. some of these are actual system settings (`emailConfigured`) and some of them are added for facilitative for development (e.g. `uiLocale` or `refreshOnTranslation`, `error`). I guess it might be better to separate these out more, but I couldn't think of a good way (/name) to organize them more.
2. I've just put all of these types as optional mostly to indicate that the app should handle situations where they are missing from the api (but don't know if that is worthwhile or not)